src/test/java/com/teragrep/pth_06/planner/bloomfilter/TableFiltersTest.java

```diff
@@ -138,7 +141,7 @@ public void testCreation() {
 
     @Test
     public void testInsertFiltersIntoCategoryTable() {
```

Comment:
please rename exception throwing test so that it describes why exception is expected

Reply:
updated test to check for values inserted into temp table and added separate exception test when temp table is not in database.